lib/rspec/core/configuration.rb

```diff
@@ -471,6 +473,9 @@ def initialize
       # Used to set higher priority option values from the command line.
       def force(hash)
         ordering_manager.force(hash)
+        [:deprecation_stream, :output_stream].each do |stream|
+          hash[stream] = prepare_stream(hash[stream]) if hash.key?(stream)
```

Comment:
Thank you. It’s my mistake, i did not intend to change the original hash. I will change the code.